assets/prototype/domain/shared/predicates/tickets/isArchived.ts

```diff
@@ -1,10 +1,16 @@
+/**
+ * Internal dependencies
+ */
+import { Ticket } from '../../../eventEditor/data/types';
+
 /**
  * @function
- * @param {Object} ticket model object
+ * @param {Object} ticket object
  * @return {boolean} true if ticket is archived
  */
-const isArchived = (ticket) => {
-	return ticket.deleted;
+const isArchived = (ticket: Ticket) => {
+	// TODO: update according to this when it's ready: https://github.com/eventespresso/event-espresso-core/issues/2089
+	return ticket['deleted'];
```

Comment:
needs typing for return value

Reply:
Added: https://github.com/eventespresso/event-espresso-core/pull/2150/commits/b8bebcf79946133a296c6ad4cccd52bddb17ea85